models/src/main/thrift/content/v1.thrift

```diff
@@ -901,6 +903,28 @@ struct ListElementFields {
     2: optional ListType type;
 }
 
+struct TimelineEvent {
+    1: required list<BlockElement> body = [];
+
+    2: optional BlockElement main;
+
+    3: optional string title;
+
+    4: optional string date;
```

Comment:
Presume the reason we are not using `CapiDateTime` here is because this is audience facing and may be formatted in many different ways?

Reply:
We need to allow timelines to include events where the date is not exactly known, where it could be a range, etc. At some point in the future I'd like to explore making this machine readable (perhaps using https://www.loc.gov/standards/datetime/). But for version 1, a string seems reasonable.